### PR TITLE
add removed files and remove jquery

### PR DESCRIPTION
**Pull request by ji3alae**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,16 +2,23 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { MainPageComponent } from './main-page/main-page.component';
 import { WawbComponent } from './maps/m1wawb/wawb.component';
+import { SpecToursComponent } from './maps/m2spectours/spectours.component';
```
Comment (akariv) [inline src/app/app-routing.module.ts]: note the unrelated changes here...

File: src/app/app.component.ts [hunk 2]
```diff
@@ -16,6 +16,6 @@ export class AppComponent {
       const html = linkRenderer.call(renderer, href, title, text);
       return localLink ? html : html.replace(/^<a /, `<a target="_blank" rel="noreferrer noopener nofollow" `);
     };
-    marked.use({renderer});
+    // marked.use({renderer});
```
Comment (akariv) [inline src/app/app.component.ts]: why?

File: src/app/main-page/main-page.component.html [hunk 3]
```diff
@@ -1,71 +1,70 @@
 <div class='viewport'>
 
-    <div class='slide' data-index='-1'>
```
Comment (akariv) [inline src/app/main-page/main-page.component.html:3]: why make any changes to the main page component?

File: src/app/app.module.ts [hunk 4]
```diff
@@ -27,6 +28,18 @@ import { InfobarComponent } from './templates/talking-heads/infobar/infobar.comp
 import { MainPageComponent } from './main-page/main-page.component';
 import { TimelineMapComponent } from './templates/timeline/timeline-map/timeline-map.component';
 import { TalkingHeadsMapComponent } from './templates/talking-heads/talking-heads-map/talking-heads-map.component';
+import { SlickCarouselModule } from 'ngx-slick-carousel';
+import { M6Component } from './maps/m6/m6.component';
+import { SeclayoutComponent } from './maps/m6/seclayout/seclayout.component';
+import { PrevDirective } from './maps/m6/directives/prev.directive';
+import { NextDirective } from './maps/m6/directives/next.directive';
+import { AccordionDirective } from './maps/m6/directives/accordion.directive';
```
Comment (akariv) [inline src/app/app.module.ts:34]: This directive was added but is not used anywhere...

File: src/app/templates/sidebar/sidebar.component.less [hunk 5]
```diff
@@ -1,187 +1,180 @@
-@import '../../../common.less';
```
Comment (akariv) [inline src/app/templates/sidebar/sidebar.component.less:1]: another unrelated file modified...

File: src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less [hunk 6]
```diff
@@ -1,32 +1,44 @@
-@import '../../../../common.less';
```
Comment (akariv) [inline src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less:1]: why was this file changed?

File: src/index.html [hunk 7]
```diff
@@ -1,8 +1,9 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <!-- Primary Meta Tags -->
-  <title>{{ title }}</title>
```
Comment (akariv) [inline src/index.html:5]: why?

File: src/styles.less [hunk 8]
```diff
@@ -1,33 +1,82 @@
...
-    font-weight: 300 800;
-    font-display: swap;
-    src: url(assets/fonts/manrope.ttf) format('opentype');
+  font-family: "Manrope";
+  font-style: normal;
+  font-weight: 300 800;
+  font-display: swap;
+  src: url(assets/fonts/manrope.ttf) format("opentype");
+}
+.text-content,
```
Comment (akariv) [inline src/styles.less:35]: do not add component specific styles here! that's what the component styles are for.

File: src/assets/img/wawb-info-icon.svg [hunk 9]
```diff
@@ -1,5 +1,5 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (akariv) [inline src/assets/img/wawb-info-icon.svg:1]: lots of unrelated image files changed here (not commenting on all of them) - please restore them.

File: src/app/maps/m6/m6.component.html [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+              <div   class="carousel thumbnail-slick" *ngFor='let groupe of images; let g = index ; let isFirst = first' [class.active]="isFirst" [attr.data-index]="g" >
+                <div class="slide-item slide-item-v" *ngFor='let image of groupe ; let i = index ; let isFirst = first ' [attr.data-index]="i">
+                  <figure >
+                      <img alt="" [src]=image.url [attr.data-index]="i+(4*g)" appSynch>
+                  </figure>
+                </div>
+              </div>
+            </div>
+
+              <div *ngIf='images?.length > 1' class="pagination">
```
Comment (akariv) [inline src/app/maps/m6/m6.component.html:50]: it's much more readable to create a pagination component which contains all the logic, instead of implementing it in 3 different directives which use document wide classes which might interfere with other components on the page.

File: src/app/maps/m6/info/info.component.html [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<!-- <div class='title'>
```
Comment (akariv) [inline src/app/maps/m6/info/info.component.html]: please remove commented out code

File: src/app/maps/m6/directives/synch.directive.ts [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+  @HostListener('click')
+  prevFunc() {
+    const slideElements = document.querySelectorAll('.slide-item-v');
+    slideElements.forEach((elem) => {
+      elem.classList.remove('active');
+    });
+
+    this.el.nativeElement.parentElement.parentElement.classList.add('active');
+    const elm = this.el.nativeElement.getAttribute('data-index');
+    const count = document.querySelectorAll('.slide-item.item').length;
```
Comment (akariv) [inline src/app/maps/m6/directives/synch.directive.ts:17]: All these manipulations using global styles are super dangerous and not the preferred way to do it in Angular. This should have been a component, using `[class.x]` bindings and `*ngFor`s to craft the elements in the list, instead of doing it 'manually' like this.

File: src/app/main-page/main-page.component.ts [hunk 13]
```diff
@@ -1,4 +1,4 @@
-import { Component, ElementRef, NgZone, OnDestroy, OnInit } from '@angular/core';
```
Comment (akariv) [inline src/app/main-page/main-page.component.ts:1]: don't make any changes to this file.

File: src/app/app.module.ts [hunk 14]
```diff
@@ -52,10 +66,22 @@ import { TalkingHeadsMapComponent } from './templates/talking-heads/talking-head
...
     TimelineMapComponent,
     TalkingHeadsMapComponent,
+    PrevDirective,
+    NextDirective,
+    M6Component,
+    SeclayoutComponent,
+    AccordionDirective,
+    SynchDirective,
+    PaginationDirective,
+    InfoComponent,
```
Comment (akariv) [inline src/app/app.module.ts:73]: Perhaps it would be a good idea to give your components some prefix if their names are too generic (e.g. this one...)

File: package.json [hunk 15]
```diff
@@ -25,7 +25,9 @@
     "dayjs": "^1.9.7",
     "mapbox-gl": "^2.1.1",
     "marked": "^1.2.7",
+    "ngx-slick-carousel": "^0.5.1",
```
Comment (akariv) [inline package.json]: is this library used in the code? couldn't find any usage of it.

**Review by akariv — CHANGES_REQUESTED** [review]
Initial review of pull request.

File: src/app/app.module.ts (hunk 4 above)
Comment (ji3alae) [reply src/app/app.module.ts:34]: they used att line 69

File: src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less (hunk 6 above)
Comment (ji3alae) [reply src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less:1]: the vs code formatter change just the quotation marks from '' to ""
@import "../../../../common.less";

File: src/app/app.module.ts (hunk 4 above)
Comment (akariv) [reply src/app/app.module.ts:34]: It's declared in line 69 - where is it actually used?

File: src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less (hunk 6 above)
Comment (akariv) [reply src/app/templates/talking-heads/talking-heads-map/talking-heads-map.component.less:1]: Then please disable automatic formatters, and either case don't include in this pull request files that are not part of your code.

**Comment by akariv** [comment]
@ji3alae please avoid resolving comments until I confirm they are actually resolved...
